tests/behat/features/asp/pegasus-unauth.feature

```diff
@@ -15,88 +15,139 @@ Feature: test ACCESS Support Pegasus Page
+    Then I should be on "https://allocations.access-ci.org/"
+    When I go to "/pegasus"
+
+    Then I should see "Setup"
+    Then I should see "The first time you logon, you need to specify what allocations you have."
+    #Then I should see an image with alt text "Login to ACCESS"
+
+    Then I should see "Single Sign On with your ACCESS ID"
+    Then I should see "All registered users with an active allocation automatically have an ACCESS Pegasus account."
+    #Alt is not working come back to this
+    #Then I should see an image with alt text "OpenOndemand instance"
+
+    Then I should see "Configure resources once"
+    Then I should see "Use Open OnDemand instance at resource providers to install SSH keys and determine location allocation ID."
+    Then I should see "Run Workflows on ACCESS"
+    #Then I should see an image with alt text "Create the workflow"
+
+    #Then I should see an image with alt text "Workflow step 1"
+    Then I should see "1. Create the workflow"
+    Then I should see "Use Pegasus API in Jupyter Notebook or use our examples"
+    #Then I should see an image with alt text "Provision compute resources"
 
-    # When I am on "/pegasus"
-    # When I follow "Learn More"
-    # Then I should get a "200" HTTP response
+    #Then I should see an image with alt text "Workflow step 2"
+    Then I should see "2. Provision compute resources"
+    Then I should see "Use HTCondor Annex tool to provision pilot jobs on your allocated ACCESS resources"
+    #Then I should see an image with alt text "Monitor the execution"
 
-    # #Then I should be on "https://access-ci.atlassian.net/wiki/spaces/ACCESSdocumentation/pages/129140407/ACCESS+Pegasus"
+    #Then I should see an image with alt text "Workflow step 3"
+    Then I should see "3. Monitor the execution"
+    Then I should see "Follow the workflow execution within the notebook or in the terminal"
+    When I click "Try it now"
+    Then I should be on "/authorize?response_type=code&scope=openid%20email%20org.cilogon.userinfo&client_id=cilogon%3A%2Fclient_id%2F57662fc94d2fcfbae1faffc582a30714&state=MDOHdQqOgYI2HK1rt-1kJkdTpIc&redirect_uri=https%3A%2F%2Faccess.pegasus.isi.edu%2Foidc&nonce=3lLLvhQ-SS0BbVPHP0jrKzIB8nM2xwT6dZemDT0SvhM"
 
-    # When I am on "/pegasus"
-    # When I follow "Pegasus Website"
-    # Then I should be on "https://pegasus.isi.edu/"
+    When I go to "/pegasus"
+    Then I should see "Tutorial Video"
+    #Then I should see an image with alt text "Video icon"
+    Then I should see "A step by step tutorial video"
+    When I click "Watch"
+    Then I should be on "/watch?reload=9&v=4wiyLFGwJg4"
+
+
+    When I go to "/pegasus"
+    Then I should see "Documentation"
+    #Then I should see an image with alt text "Documentation icon"
+    Then I should see "Detailed documentation about"
+    When I click "Read"
+    Then I should be on "/wiki/spaces/ACCESSdocumentation/pages/201752577/ACCESS+Pegasus"
```

Comment:
The Confluence wiki urls change, so we should not test for this. Try testing for a 200 response instead.